front/src/applications/operationalStudies/hooks/useSetupItineraryForTrainUpdate.ts

```diff
@@ -71,9 +70,7 @@ const computeBasePathSteps = (trainSchedule: TrainScheduleResult) =>
       ...stepWithoutSecondaryCode,
       ch: 'secondary_code' in step ? step.secondary_code : undefined,
       name,
-      arrival: arrival
-        ? addDurationToIsoDate(trainSchedule.start_time, arrival).substring(11, 19)
-        : arrival,
+      arrival,
```

Comment:
To keep track at what kind of arrival we have here ?

```suggestion
      arrival, // as ISODurationString
```

Reply:
sure done